module-2/backend/week_one.md

```diff
@@ -5,30 +5,58 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+  Get, Post, Redirect, Render, Put
 2. What is Sinatra?
+  A local hosting service to test http functionality.
 4. What is MVC?
+  Models, views, controllers
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+  So that other coders/people can look in to our app and make sense of it.
 6. What types of variables are accessible in our view templates without explicitly passing them?
+  instance variables, in that specific block
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    ** @name = 'Mr. Ed'

```

Comment:
this is passing the instance variable `@name`, not the local variable. how would we pass the local variable through?